.github/workflows/retag_image.yaml

```diff
@@ -0,0 +1,32 @@
+name: retag-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      image-name:
+        description: 'Image name'
+        type: string
+        required: true
+      existing-tag:
+        description: 'Existing tag'
+        type: string
+        required: true
+      new-tag:
+        description: 'New tag'
+        type: string
+        default: "latest"
+
+jobs:
+  tag_image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
For consistency I'd even add the checkout action to the custom action itself, as that is what we did in the other custom actions in .github-Repo as well. But I am not 100% sure if this is best practice.

Reply:
I left it there as this isn't required for the workflow but only for accessing the action and can be removed after we moved the action to .github.